packages/dds/map/src/directory.ts

```diff
@@ -826,7 +831,8 @@ export class SharedDirectory
 					const localValue = this.makeLocal(
 						key,
 						currentSubDir.absolutePath,
-						serializable,
+						// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+						parseHandles(serializable, this.serializer),
```

Comment:
Maybe instead, replace `readAndParse` (which does `JSON.parse`) with `readBlob` + `serializer.parse`? That's preferable because then it's consistent throughout this code - handles are not encoded. Check out `loadCore` in other DDSes, I glanced at Matrix and CRC for reference.